cmd/keymasterd/main.go

```diff
@@ -1736,22 +1733,19 @@ func init() {
 	prometheus.MustRegister(externalServiceDurationTotal)
 	prometheus.MustRegister(certDurationHistogram)
 	tricorder.RegisterMetric(
-		"keymaster/externalServiceDuration/LDAP",
+		"keymaster/external_service_duration/LDAP",
 		tricorderLDAPExternalServiceDurationTotal,
 		units.Millisecond,
-		//tricorder.None,
 		"Time for external LDAP server to perform operation(ms)")
 	tricorder.RegisterMetric(
-		"keymaster/externalServiceDuration/VIP",
+		"keymaster/external_service_duration/VIP",
 		tricorderVIPExternalServiceDurationTotal,
 		units.Millisecond,
-		//tricorder.None,
 		"Time for external VIP server to perform operation(ms)")
 	tricorder.RegisterMetric(
-		"keymaster/externalServiceDuration/Storage",
+		"keymaster/external_service_duration/Storage",
```

Comment:
The convention is hyphens, not underscores. Also, "Storage" should be lowercase.